subprojects/core/src/main/java/org/gradle/api/internal/tasks/AbstractTaskOutputPropertySpec.java

```diff
@@ -66,8 +66,8 @@ public String toString() {
         return getPropertyName() + " (OUTPUT)";
     }
 
-    public Class<? extends FileCollectionSnapshotter> getSnapshotter() {
-        return GenericFileCollectionSnapshotter.class;
+    public Class<? extends PropertyNormalizer> getSnapshotter() {
```

Comment:
Can we change this method to `getNormalizer`? Or can we even get rid of it completely for output properties?

Reply:
Renamed. We still need a snapshotter, so I think it makes sense to keep it for output properties, too.